websites/M/Mahara-Tech/metadata.json

```
@@ -0,0 +1,40 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "605894319408283678",
+		"name": "eljooker"
+	},
+	"service": "Mahara-Tech",
+	"description": {
+		"en": "information technology learning platform"
+	},
+	"url": "maharatech.gov.eg",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/SjtqzAF.png",
+	"thumbnail": "https://i.imgur.com/dmO42Up.png",
+	"color": "#FFFFFF",
```

The red color may be better here?
```suggestion
	"color": "#8d1f28",
```